### PR TITLE
Simple intergration thisvui with vue cli

### PR DESCRIPTION
**Pull request by musasutisna**
figure more configuration in the future.

### CODE REVIEW

File: generator/index.js [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+  })
+
+  // import packages thisvui to main.js
+  api.injectImports(api.entryFile, `import ThisVui from 'thisvui'`)
+
+  if (options.addStyle === 'css') {
+    api.injectImports(api.entryFile, `import 'thisvui/dist/thisvui.min.css'`)
+  } else if (options.addStyle === 'scss') {
+    api.render('./templates/style')
+    api.injectImports(api.entryFile, `import '@/assets/scss/thisvui.scss'`)
```
Comment (ncanatta) [inline generator/index.js]: Think this line should be 'thisvui/dist/sass/thisvui.scss'

**Review by ncanatta — CHANGES_REQUESTED** [review]
hey @mu545 excellent work! Thank you so much!

I leave some points to review:

- While testing the plugin, I found a small bug when choosing **scss** as an option.
- When **scss** option is chosen, we should also add **"node-sass"** and **"sass-loader"** as dependencies.

File: generator/index.js (hunk 1 above)
Comment (musasutisna) [reply generator/index.js]: I am sorry about that.
now I have added sass loader on extend package.

Are you sure to place the source style on the outside template directory?

File: generator/index.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ module.exports = (api, options) => {
     })
 
     api.render('./templates/style')
-    api.injectImports(api.entryFile, `import '@/assets/scss/thisvui.scss'`)
+    api.injectImports(api.entryFile, `import 'thisvui/dist/sass/thisvui.scss'`)
```
Comment (ncanatta) [inline generator/index.js:25]: Hey @mu545 thank you so much, i am kind new in writing plugins for vue-cli, do you recommend handling source styles inside the template folder?

Comment (musasutisna) [reply generator/index.js:25]: Handling source style inside template folder will make the source style can be customized by users.
Import all main of sources style be need in '@/assets/scss/thisvui.scss' so it allows a user to customize.

Comment (ncanatta) [reply generator/index.js:25]: I see, thanks for the help and insight, i'll create a issue to handle this.

**Pull request merged**